shell/packages/sandstorm-ui-powerbox/powerbox.html

```diff
@@ -0,0 +1,80 @@
+<template name="powerboxRequest">
+  <h4>Powerbox request</h4>
+  {{#if showWebkeyInput}}
+    <form id="powerbox-request-form">
+      <label>Please input an api token: </label><input name="token" type="text" id="powerbox-request-input"><br>
+      <button class="submit">Import token</button>
+    </form>
+    {{#if webkeyError}}
+    <div class="error">{{webkeyError}}</div>
+    {{/if}}
+  {{else}}
+    <h4>Connect this grain with:</h4>
```

Comment:
I feel like this wording may be confusing in common use cases. E.g. when sharing through Rocket.Chat, users wouldn't normally think of what they're doing as "connecting". Maybe "attaching" makes sense, or maybe this should just say "Choose a grain:"? I think it's well-enough implied from the context that the currently-open app is making the request and will get access -- similarly, the browser's file-open dialog doesn't explicitly say that you're granting the web page access to the file.